docs/features/software-catalog/descriptor-format.md

```diff
@@ -383,8 +383,6 @@ spec:
   lifecycle: production
   owner: artist-relations-team
   system: artist-engagement-portal
-  providesApis:
```

Comment:
Should this have disappeared?

Reply:
Hmm you are right. For some reason I wanted to remove this 🤔 😆 No it has to stay!